backend/de.metas.material/planning/src/main/java/de/metas/material/planning/event/SupplyRequiredDecreasedHandler.java

```diff
@@ -80,8 +106,48 @@ public void handleEvent(@NonNull final SupplyRequiredDecreasedEvent event)
 
 		if (remainingQtyToHandle.signum() > 0)
 		{
-			Loggables.withLogger(log, Level.WARN).addLog("Could not decrease the qty for order {}. Qty left: {}", descriptor.getOrderId(), remainingQtyToHandle);
-			//TODO add notification here
+			Loggables.withLogger(log, Level.WARN).addLog("Could not decrease the qty for event {}. Qty left: {}", event, remainingQtyToHandle);
+			sendNotification(descriptor, remainingQtyToHandle);
 		}
 	}
+
+	private void sendNotification(final SupplyRequiredDescriptor descriptor, final Quantity unhandledQty)
+	{
+		final TableRecordReference recordReference = getTableRecordReferenceFromEventDescriptor(descriptor);
+		if (recordReference == null)
+		{
+			return;
+		}
+		final PO po = InterfaceWrapperHelper.getPO(recordReference.getModel(PlainContextAware.newWithThreadInheritedTrx(), Object.class));
```

Comment:
hmm, why not using de.metas.material.event.commons.EventDescriptor#userId (not sure what user is there)?

i am kind of reticient 
* about using getModel method which is deprecated
* loading the PO model out of the blue
* using the UpdatedBy field

IMHO we shall try using sometihng more stable like de.metas.material.planning.ProductPlanning#plannerId

Reply:
as discussed moved to use:
- ProductPlanning#plannerId - is available and is a regular user
- C_Order.SalesRep_ID - if available & regular user
- C_Order.UpdatedBy - if regular user
- OrgInfo#SupervisorId - if regular user, only as dead-letter recipient.